### PR TITLE
Switch COS validation tests to using the LTS images

### PR DESCRIPTION
**Pull request by yguo0905**
- There are two LTS versions `cos1` and `cos2`. The new tests suites will be
  `cos1-k8sbeta`
`cos1-k8sstable1`
`cos1-k8sstable2`
`cos1-k8sstable3`
`cos2-k8sbeta`
`cos2-k8sstable1`
`cos2-k8sstable2`
`cos2-k8sstable3`
- I'd like to have each of the above in its own tab, so the entire cos validation tab is a top-level one.
- This shouldn't affect any release jobs.

### CODE REVIEW

File: testgrid/cmd/configurator/config_test.go [hunk 1]
```diff
@@ -38,6 +38,7 @@ type SQConfig struct {
 var (
 	companies = []string{
 		"canonical",
+		"cos",
```
[the remark this answers is not in the document]
Comment (BenTheElder) [reply testgrid/cmd/configurator/config_test.go]: nevermind, I see that you're trying to do a group of dashbaords for cos itself which won't nest ...

@michelle192837 what do you think about nested dashboard groups? is that something we want to support?

**Comment by krzyzacy** [comment]
curious how did you handle testgrid config? :-)

/lgtm
/hold

**Comment by yguo0905** [comment]
> curious how did you handle testgrid config? :-)

Manually. If we can separate the testgrid config into different pieces, we can probably generate them too.

Is this ready to go?

**Comment by krzyzacy** [comment]
sure
yeah we technically have multi-config support, but haven't migrated them yet.
/hold cancel

**Pull request merged**